### DISCUSSION TITLE
中文支持的怎么样

### DISCUSSION THREAD

**Discussion by kucoonit** [raingggg/coctohug#4] [Q&A] [answered]
有装中文的没，支持的怎么样？

**Comment by marisome** [comment, marked as answer]
挺好的，比一些教科书翻译的质量好多了。能多支持一些币种就好了，64G内存已经下单了,几十个一起挖，希望能快速回本

**Reply by raingggg** [reply]
Welcome! Let me know if there are any improvements needed on the Chinese translation!
